app/static/pages/app.js

```diff
@@ -13,8 +13,8 @@
 // limitations under the License.
 
 var oppiaFoundationWebsite = angular.module(
-  'oppiaFoundationWebsite', ['ngMaterial', 'ngMessages', 'ngRoute', 'duScroll',
-    'ui.bootstrap', 'headroom']);
+  'oppiaFoundationWebsite', ['angular-matchmedia', 'ngMaterial', 'ngMessages',
```

Comment:
I don't think you need to add a 3rd-party lib for matchmedia -- just make a small service on our end, similar to what we do in oppia (windowDimensionsService).

Reply:
Yep, I agree with not using a 3rd party lib. Replaced 3rd party angular-matchmedia with WindowDimensionsService!